src/web/waiters/ControlsWaiter.mjs

```diff
@@ -51,6 +51,18 @@ class ControlsWaiter {
         }
     }
 
+    /**
+     * Checks or unchecks the Auto Bake checkbox with "Enter"
+     * @param {Event} ev
+     */
+    autoBakeKeyboardHandler(ev) {
```

Comment:
Whilst this solves the problem, a more accessible solution might be to replace the `<span />` check implementation with a proper check input box?  That way, we would get accessibility by default.

Reply:
The change I have made adds the `enter` key. Also there is already a input checkbox there. 
The problem here is that when I use the space bar initially  without my changes , the focus ends up being stolen which the prevents me reverting that change.